products/add-data-to-grid-datastores/datastore-faq.md

```diff
@@ -22,7 +25,7 @@ Grid does not charge for storing data. However, there is a limit of 25 datastore
 
 ## Does Grid have a limit on DataStore size?
 
-Yes Grid DataStore max size is 1.5 TB. For security reasons, we only allow you to upload data for 12 hours continuously. If your data is too big to upload in a 12 hour period, it will stop uploading. Initiating `grid datastore create <args...>` again will resume the upload.
+Yes Grid DataStore max size is 1.5 TB. For security reasons, we only allow you to upload data for 24 hours continuously. If your data is too big to upload in a 24 hour period, it will stop uploading. Initiating `grid datastore create <args...>` again will resume the upload.
 
```

Comment:
we should clarify that the max size is 1.5 TB via the CLI only. via the UI it's actually 1G @oojo12